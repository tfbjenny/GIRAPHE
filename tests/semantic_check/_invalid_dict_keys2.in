dict<int> a = {"a": 1, "b":2};
list<float> b = a.keys();