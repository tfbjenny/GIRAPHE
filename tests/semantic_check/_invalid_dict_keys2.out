illegal assignment list<float> = list<null> in b = function call a.keys
